src/EasyAbp.Abp.Sms.TencentCloud/EasyAbp/Abp/Sms/TencentCloud/SmsSendingException.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace EasyAbp.Abp.Sms.TencentCloud
+{
+    public class SmsSendingException : ApplicationException
```

Comment:
Can rename to `SmsSendingFailedException`.

Can use `: BusinessException` and provide a Code like `EasyAbp.Abp.Sms.TencentCloud:SmsSendingFailed`. If you do this change, please don't forget to add localizations for it: https://docs.abp.io/en/abp/latest/Exception-Handling#exception-localization.